edge-apps/clock/screenly.yml

```diff
@@ -0,0 +1,9 @@
+app_id: 01H55NQ0290YC714DF36KQ9PN7
+root_asset_id: ''
+user_version: 'v1.0.0'
+revision: 1
+description: 'Displays the current time'
+icon: ''
+author: 'Screenly, Inc.'
+homepage_url: ''
+settings: {}
```

Comment:
Add the following settings:

* "override_timezone": overrides the timezone based on location
* "disable_analytics": Disable Sentry and GA